doc/api/http.md

```diff
@@ -3187,6 +3187,76 @@ Removes a header that is queued for implicit sending.
 outgoingMessage.removeHeader('Content-Encoding');
 ```
 
+### `outgoingMessage.writeFile(file[, options], callback)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development.
+
+* `file` {string|URL|number} file path, URL or descriptor.
```

Comment:
Probably worthwhile for this to support `FileHandle` also.

Reply:
There are some issues with filehandle I'd like to sort out before that.

Reply:
I would recommend against using that api in general due to issue with how the reference counting behaves.